operators/endpointmetrics/controllers/observabilityendpoint/match_evaluator.go

```diff
@@ -3,9 +3,8 @@
 package observabilityendpoint
 
 import (
+	"golang.org/x/exp/slices"
```

Comment:
Why not use the `slices` package from the standard library?

Reply:
So, I'm not entirely sure how this repo is built, and the Dockerfiles seem to use 1.20. `slices` was added to stdlib in 1.21 I think